base-application.gradle

```diff
@@ -33,6 +33,12 @@ android {
             dimension "dependencyVersions"
         }
     }
+
+    sourceSets {
+        unityIAP {
+            java.srcDirs = ['src/latestDependencies/java']
+        }
+    }
```

Comment:
Is this needed?

Reply:
technically not yet, no... though planning to split out the API testers and sample app soon.